Library/Homebrew/lock_file.rb

```diff
@@ -54,7 +55,7 @@ def lock
             raise OpenFileChangedOnDisk
           end
 
-          @lockfile = lockfile
+          @lockfile = T.let(lockfile, T.nilable(File))
```

Comment:
Is the `T.let` here needed, given that you already do it in `initialize`?

Reply:
@carlocab @botantony Shouldn't ever be needed more than once per class/file.